.kokoro/build.sh

```diff
@@ -42,12 +53,19 @@ if [[ ! -z "${GOOGLE_APPLICATION_CREDENTIALS}" && "${GOOGLE_APPLICATION_CREDENTI
     export GOOGLE_APPLICATION_CREDENTIALS=$(realpath ${KOKORO_GFILE_DIR}/${GOOGLE_APPLICATION_CREDENTIALS})
 fi
 
+# units-java8 uses both JDK 11 and JDK 8. We ensure the generated class files
+# are compatible with Java 8 when running tests.
+if [ ! -z "${JAVA8_HOME}" ]; then
+  setJava "${JAVA8_HOME}"
+  mvn -version
+fi
+
 RETURN_CODE=0
 set +e
 
 case ${JOB_TYPE} in
 test)
-    mvn test -B \
+    mvn test -B -V \
```

Comment:
What do we need the version of mvn for - does it tie to native image compilation in some way?

Reply:
This clarifies which versions of JDK it's running. We're running JDK 11 to compile and JDK 8 to run tests.